charts/partials/templates/_proxy.tpl

```diff
@@ -68,6 +68,14 @@ env:
 - name: LINKERD2_PROXY_TAP_SVC_NAME
   value: linkerd-tap.$(_l5d_ns).serviceaccount.identity.$(_l5d_ns).$(_l5d_trustdomain)
 {{ end -}}
+{{ if .Proxy.Trace -}}
+{{ if .Proxy.Trace.CollectorSvcAddr -}}
+- name: LINKERD2_PROXY_TRACE_COLLECTOR_SVC_ADDR
+  value: {{ .Proxy.Trace.CollectorSvcAddr }}
+- name: LINKERD2_PROXY_TRACE_COLLECTOR_SVC_NAME
+  value: {{ .Proxy.Trace.CollectorSvcAccount }}.serviceaccount.identity.$(_l5d_ns).$(_l5d_trustdomain)
```

Comment:
I found it a bit hard to follow that this variable contains both the service account name and namespace.  I think it would be more clear if there was a template variable for the entire identity string and the value for the whole identity string was composed in the injector code *OR* if there were two separate template variables for service account name and service account namespace.

Reply:
For some reasons, I thought that `<serviceaccount>.<ns>` is the fully qualified name of a service account, but turns out it's something like `<group>:<ns>:<serviceaccount>`. I'm gonna keep the `.serviceaccount.identity.$(_l5d_ns).$(_l5d_trustdomain)` as-is to remain consistent with the rest of the `*_SVC_NAME` in this template, and might just change the `CollectorSvcAccount` variable name to something more reflective. Any suggestions on variable name?